### DISCUSSION TITLE
Welcome to ha_solis_modbus Discussions!

### DISCUSSION THREAD

**Discussion by fboundy** [fboundy/ha_solis_modbus#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bloodygoodbloke** [comment]
HI can you write a NOOB guide when you get a chance.
IE what to install and what to do to get it all working

**Comment by TikTok90** [comment]
Does this work with the DLS-W sticks - and are there any drawbacks?  I'd really like to get better control of my inverter as I'm getting really hacked off with having to go through all the necessary steps in the app to make changes to my charge/discharge times to make best use of the power-ups.

**Comment by fboundy** [comment]
I don’t think so. You need the SolarMan integration for that

**Comment by nieckx** [comment]
The connection I have is serial RS485 to USB. 
I tried it as per: https://homeassistant-solax-modbus.readthedocs.io/en/latest/installation/
No success to get it working, but I'm new to this...

**Reply by jsa061** [reply]
Hi, first of all - nice project! Looks like you have done some serious work here. 

Small introduction: 

Working as a programmer and like technical things like I would Imagine all others here 😄 . I have 10kw solar setup with a Solis S5-GPR3P10K-LV  inverter. Just bought the S3-WIFI-ST and now I am wondering if that was a mistake 😅 

Could you comment on why the S3-WIFI-ST will not work? 🙏

**Reply by Paulholloway** [reply]
The reasoning is better detailed here:

[https://community.home-assistant.io/t/getting-data-from-solis-inverter/302189/82](https://community.home-assistant.io/t/getting-data-from-solis-inverter/302189/82)

And

[https://community.home-assistant.io/t/solis-integration-for-gen-3-data-logging-stick-s3-wifi-st/437427/6]()

You can get data from the S3 into HA but it's via SolisCloud, at at 5min intervals, read only etc.

**Reply by jsa061** [reply]
Thanks for the reply. That put me in the right direction. Seems like it is possible to use he S3-WIFI-ST but you need to flash it with a ESPHome image (. https://github.com/hn/ginlong-solis?tab=readme-ov-file#solis-s3-wifi-data-logging-stick-3rd-gen )
I bought a USB to Serial IC adapter for the flashing. Probably going to take some time before I get but will definitely try 😄
